room_assistance/indico_room_assistance/templates/emails/creation_email_to_assistance.html

```diff
@@ -11,7 +11,7 @@
         {{ requested_by.full_name }} requested assistance to setup {{ event.room.full_name }} before the start of the event.
     </p>
     <div>
-        <strong>Event:</strong> {{ event.title }}<br>
+        <strong>Event:</strong> {{ event.title }} ({{ event.external_url }})<br>
```

Comment:
It's an HTML email - why not use an HTML link as well? (could just make the title a link)

Reply:
Oh, true.